shell/src/main/java/alluxio/cli/fs/command/GetFaclCommand.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli.fs.command;
+
+import alluxio.AlluxioURI;
+import alluxio.cli.CommandUtils;
+import alluxio.client.file.FileSystem;
+import alluxio.client.file.URIStatus;
+import alluxio.exception.AlluxioException;
+import alluxio.exception.status.InvalidArgumentException;
+
+import org.apache.commons.cli.CommandLine;
+
+import java.io.IOException;
+
+import javax.annotation.concurrent.ThreadSafe;
+
+/**
+ * Displays ACL info of a path.
+ */
+@ThreadSafe
+public final class GetFaclCommand extends AbstractFileSystemCommand {
+  /**
+   * @param fs the filesystem of Alluxio
+   */
+  public GetFaclCommand(FileSystem fs) {
+    super(fs);
+  }
+
+  @Override
+  public String getCommandName() {
+    return "getfacl";
+  }
+
+  @Override
+  protected void runPlainPath(AlluxioURI path, CommandLine cl)
+      throws AlluxioException, IOException {
+    URIStatus status = mFileSystem.getStatus(path);
+    System.out.println("# file: " + status.getPath());
+    System.out.println("# owner: " + status.getOwner());
+    System.out.println("# group: " + status.getGroup());
+    System.out.println(status.getAcl());
```

Comment:
Should this also start with `# acl:` or are there too many new lines?

Reply:
I am just following the standard `getfacl` command: https://linux.die.net/man/1/getfacl